### DISCUSSION TITLE
connect to local port

### DISCUSSION THREAD

**Discussion by maziac** [community/community#30836] [Codespaces]
How can I connect from code spaces running in the browser to a port on my local PC?

**Comment by ihnatenkoo** [comment]
Hello, look here friend:)
[https://docs.github.com/en/codespaces/developing-in-codespaces/forwarding-ports-in-your-codespace](https://docs.github.com/en/codespaces/developing-in-codespaces/forwarding-ports-in-your-codespace)

**Comment by maziac** [comment]
No, that's not what I'm looking for.
I mean vice versa.
A server is running on my local PC and I want to connect from code spaces running in the browser.
